android/src/main/java/com/swmansion/reanimated/layoutReanimation/SharedTransitionManager.java

```diff
@@ -30,6 +30,7 @@ public class SharedTransitionManager {
   private final Map<Integer, View> mCurrentSharedTransitionViews = new HashMap<>();
   private View mTransitionContainer;
   private final List<View> mRemovedSharedViews = new ArrayList<>();
+  private final Set<Integer> mShouldBeHidden = new HashSet<>();
```

Comment:
`mShouldBeHidden` sounds like a boolean flag, not like a set of view(tags).
```suggestion
  private final Set<Integer> mHiddenViewTags = new HashSet<>();
```
perhaps? Or maybe
```suggestion
  private final Set<Integer> mViewTagsToHide = new HashSet<>();
```